psi4/src/psi4/ccdensity/get_moinfo.cc

```diff
@@ -32,8 +32,8 @@
 */
 #include <cstdio>
 #include <cstdlib>
-#include <string.h>
-#include <string.h>
+#include <cstring>
+#include <cstring>
```

Comment:
Did you mean to include `<string>` instead of two includes of `<cstring>`?  No harm in including a header twice, of course, but I just wanted to check that it's not a typo that will cause problems

Reply:
I'll fix it!